hr_holidays_imposed_days/README.rst

```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+================
+Imposed holidays
+================
+
+Allows to manage imposed holidays.
+
+This can be used to manage periods where the company is closed for everyone,
+such as end of year celebration. The HR team can use this to easily create
```

Comment:
In my opinion end of year celebration is not a good example since this module should be used to handle situations like a company closing the whole August, for example, or some weeks. The 1st of January is a public holiday and should be declared using the `hr_holidays_public` module.
Maybe someone wants to use it to handle also these events and doesnt want to install the public holidays module but i would write the other case in the readme.

Reply:
I think it means, when the company closes for a week or two, not only the holidays.